source/includes/roles/_10-index.md.erb

```diff
@@ -0,0 +1,93 @@
+## Get all roles
+
+```shell
+$ curl 'https://ci.example.com/go/api/admin/security/roles' \
```

Comment:
Not related, but do we have plans of filtering based on `gocd` or `plugin` role?